network/basics/uri.py

```diff
@@ -369,7 +368,6 @@ def main():
             body_format = dict(required=False, default='raw', choices=['raw', 'json']),
             method = dict(required=False, default='GET', choices=['GET', 'POST', 'PUT', 'HEAD', 'DELETE', 'OPTIONS', 'PATCH']),
             return_content = dict(required=False, default='no', type='bool'),
```

Comment:
I thought so too until I looked at the linked issue which seems to add it there.